src/outputErlangCode.ml

```diff
@@ -151,6 +151,20 @@ let stringify_format_element = function
       in
       (1, Printf.sprintf "~%s%s" s ch)
 
+let escape_character c =
+  match Uchar.to_int c with
+  | 10 -> [ Uchar.of_char '\\'; Uchar.of_char 'n' ]
+  | 13 -> [ Uchar.of_char '\\'; Uchar.of_char '\r' ]
+  | 9  -> [ Uchar.of_char '\\'; Uchar.of_char '\t' ]
```

Comment:
Was the implementation here intended to be the following?

```ocaml
  | 13 -> [ Uchar.of_char '\\'; Uchar.of_char 'r' ]
  | 9  -> [ Uchar.of_char '\\'; Uchar.of_char 't' ]
```

Reply:
Correct, good find! I'll push this change in a sec.